torch/_C/__init__.pyi.in

```diff
@@ -208,6 +208,7 @@ def _jit_override_can_fuse_on_gpu(override: _bool): ...
 def _jit_set_texpr_fuser_enabled(enable: _bool): ...
 def _jit_set_te_must_use_llvm_cpu(use_llvm: _bool): ...
 def _jit_set_nvfuser_enabled(enable: _bool) -> _bool: ...
+def _jit_cat_wo_conditionals(optimize_cat: _bool): ...
```

Comment:
Isn't this file autogenerated? I don't remember ever touching it manually

Reply:
Nope. There are lots of commits to this file. There is another file `__init__.pyi` which is auto-generated I think.